lib/preview-web/src/PreviewWeb.tsx

```diff
@@ -75,6 +82,10 @@ export class PreviewWeb<TFramework extends AnyFramework> {
 
   previousCleanup: StoryCleanupFn;
 
+  abortController: AbortController;
```

Comment:
NOTE: it is possible to have a render "cycle" running for more than one story at once (in docs mode). We've informally said we won't allow more than one play function at once, but that isn't enforced anywhere.

I'm not sure if an `abortController` singleton is a great idea given the above, unless we actively enforce it (e.g. throw if we see an unfinished controller when starting a `play()`?)